touchforms/formplayer/static/formplayer/script/fullform-ui.templates.js

```diff
@@ -0,0 +1,48 @@
+this["JST"] = this["JST"] || {};
+
+this["JST"]["formplayer/templates/formplayer/fullform-ui/form.html"] = function(obj) {
+obj || (obj = {});
+var __t, __p = '', __e = _.escape;
+with (obj) {
+__p += '<div>\n    <h1 id="title"></h1>\n    <div id="form"></div>\n    <input id="submit" type="submit" value="Submit" />\n</div>\n';
```

Comment:
agreed, and that does sound awful. a separate build does sound reasonable to me. it may even be wise to just put it into our fabfile since touchforms is bit more dependent on HQ?